x/ecocredit/server/core/bridge.go

```diff
@@ -21,10 +24,28 @@ func (k Keeper) Bridge(ctx context.Context, req *core.MsgBridge) (*core.MsgBridg
 
 	sdkCtx := types.UnwrapSDKContext(ctx)
 	for _, credit := range req.Credits {
+		batch, err := k.stateStore.BatchTable().GetByDenom(ctx, credit.BatchDenom)
+		if err != nil {
+			return nil, err // we already know the batch exists from Cancel
+		}
+
+		// if no batch contract entry is found then we error because we only support
+		// bridging credits from credit batches that were created as a result of a
+		// bridge operation (i.e. only previously bridged credits)
+		batchContract, err := k.stateStore.BatchContractTable().Get(ctx, batch.Key)
+		if err != nil {
+			if err == ormerrors.NotFound {
```

Comment:
nit: could use the `IsNotFound` function like we do elsewhere

Reply:
https://github.com/regen-network/regen-ledger/pull/1226/commits/0670d501dd912028814092107971d3f5c136e6b4